src/copytrade/copytrade.es6

```diff
@@ -24,6 +24,14 @@ const form_error_messages = {
 
 const getStorageName = () => `copyTrade_${getLoggedInUserId()}`;
 
+const mapCodeToApiCode = (selected_trade_types, all_trade_types) =>
```

Comment:
one more thing: can we add a check for if the `selected_trade_types` array is empty